pkgs/build-support/fetchnextcloudapp/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, pkgs, ... }:
```

Comment:
Please don't use `pkgs` here. Instead you can specify `gnutar`, `findutils` etc. explicitly. This also has the benefit that one could override these dependencies like this:

```
fetchNextcloudApp.override { findutils = myfindutis; }
```